packages/constants/tsconfig.json

```diff
@@ -1,8 +1,14 @@
 {
   "extends": ["@leather.io/tsconfig-config/tsconfig.base.json"],
-  "include": ["**/*", ".*.ts"],
+  "include": ["src/**/*", "src/.*.ts"],
   "exclude": ["./dist/"],
   "compilerOptions": {
-    "outDir": "./dist"
-  }
+    "outDir": "./dist",
+    "isolatedDeclarations": true
```

Comment:
I don't think it's worth having a different config in a single package, especially if it requires disabling the relevant rule in the eslint config, which is the reason we decided against it to begin with when we were discussing it at length.